### PR TITLE
Feature addstatus

### PR DESCRIPTION
**Pull request by ShahinaKP**
@nishintn 

Done the add status component layout
Populated the Date select field, Projects select field, Activity type select field and the time spend select field

Please review and merge the branch

### CODE REVIEW

File: src/app/components/AddDailyStatusComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  constructor(props) {
+    super(props);
+
+    // Populate the last 7 days dates
+    let datesArr = [];
+    let curr = new Date; // get current date
+    let first = curr.getDate(); // First day is the day of the month - the day of the week
+    let last = first - 6; // last day is the first day + 6
+
+    let firstday = new Date(curr.setDate(first));
```
Comment (mad-nishin) [inline src/app/components/AddDailyStatusComponent.jsx]: 15 and 11 are same.

File: src/app/components/AddDailyStatusComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      <option key={activity}>{activity}</option>
+    );
+
+    const hours = ["00", "01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"],
+      hoursSpend = hours.map((hour) =>
+        <option key={hour}>{hour}</option>
+      );
+    const minutes = ["00", "05", "30", "45"],
+      minutesSpend = minutes.map((minutes) =>
+        <option key={minutes}>{minutes}</option>
```
Comment (mad-nishin) [inline src/app/components/AddDailyStatusComponent.jsx]: Try using a mapping function that accepts a data and key and use it to populate all your drop downs.

File: src/app/components/AddDailyStatusComponent.jsx (hunk 1 above)
Comment (ShahinaKP) [reply src/app/components/AddDailyStatusComponent.jsx]: Removed unwanted codes

File: src/app/components/AddDailyStatusComponent.jsx (hunk 2 above)
Comment (ShahinaKP) [reply src/app/components/AddDailyStatusComponent.jsx]: Used a single function to map options for select

**Pull request merged**